docs/glossary.md

```diff
@@ -0,0 +1,26 @@
+| accuracy mode            | In accuracy mode, the app measures the throughput and the accuracy.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| LoadGen (Load Generator) | The LoadGen is a traffic generator for MLPerf Inference that loads the SUT and measures performance. Its behavior is controlled by a configuration file it reads at the start of the run. The LoadGen produces the query traffic according to the rules of the previously described scenarios (i.e., single-stream, multistream, server, and offline). Additionally, it collects information for logging, debugging, and postprocessing the data. It records queries and responses from the SUT, and at the end of the run, it reports statistics, summarizes the results, and determines whether the run was valid.<br>Source: <https://arxiv.org/pdf/1911.02549.pdf> |
+| single stream scenario   | The single-stream scenario represents one inference-query stream with a query sample size of 1, reflecting the many client applications where responsiveness is critical. An example is offline voice transcription on Google’s Pixel 4 smartphone. To measure performance, we inject a single query into the inference system; when the query is complete, we record the completion time and inject the next query. The metric is the query stream’s 90th-percentile latency.<br>Source: <https://arxiv.org/pdf/1911.02549.pdf>                                                                                                                                       |
+| offline scenario         | The offline scenario represents batch-processing applications where all data is immediately available and latency is unconstrained. An example is identifying the people and locations in a photo album. For this scenario, we send a single query that includes all sample-data IDs to be processed, and the system is free to process the input data in any order. Similar to the multistream scenario, neighboring samples in the query are contiguous in memory. The metric for the offline scenario is throughput measured in samples per second.<br>Source: <https://arxiv.org/pdf/1911.02549.pdf>                                                               |
+
+**Helpful links**:
+
+* <https://mlcommons.org/en/inference-mobile-11>
+* <https://github.com/mlcommons/inference_policies/blob/master/inference_rules.adoc>
```

Comment:
Could you add a short description for these links, please?

Reply:
Fixed.